<!DOCTYPE html>
<html>
	<head>
		<title>Welcome to my website</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
		
				
		<style>
		* {
  		box-sizing: border-box;
			}

		/* Create two unequal columns that floats next to each other */
		.column {
  		float: left;
  		padding: 10px;
  		}

		.left {
  		width: 30%;
		}

		.right {
 		width: 70%;
		}

		/* Clear floats after the columns */
		.row:after {
  		content: "";
  		display: table;
  		clear: both;
		}
		
		img {
			border-radius: 50%;
		}
		

		</style>
	
	</head>
	<body>
		<nav>
    		<ul>
        		<li><a href="/">About</a></li>
	        	<li><a href="/research">Research</a></li>
        		<li><a href="/publications">Publications</a></li>
        		<li><a href="/cv">CV</a></li>
    		</ul>
		</nav>
		
		<div class="about-border"></div>
		
		<div class="row">
  			<div class="column left">
   		 	<p> </p>
				<p> </p>
				<p> </p>
				<img src="https://scontent-sin6-1.xx.fbcdn.net/v/t1.6435-9/48386100_2115914461785479_4985147271789150208_n.jpg?_nc_cat=111&ccb=1-7&_nc_sid=127cfc&_nc_ohc=bVoIJztRYzAQ7kNvgFNEPu2&_nc_zt=23&_nc_ht=scontent-sin6-1.xx&_nc_gid=Arogk20wHqYR8YJTqyj8gt6&oh=00_AYB0pKXPZwd3fAT66_NwNdL4mBy9cuvrvaeIZgld0Neduw&oe=677621E0" alt="Avatar" width="155" height="155">
  				
				<p> Daniel Muliaditan </p>
				<p> Postdoctoral Fellow at the Genome Institute of Singapore (GIS) </p>
				
				<a href="https://www.linkedin.com/in/daniel-muliaditan-bb7769b2/">
			<img border="0" alt="LinkedIn" src="https://www.pngfind.com/pngs/m/53-534062_linkedin-for-lawyers-property-icon-circle-hd-png.png" width="30" height="30">
			</a> 
				<a href="https://twitter.com/dmuliaditan">
			<img border="0" alt="Twitter" src="https://www.pngfind.com/pngs/m/60-606593_icons-clipart-twitter-twitter-logo-png-transparent-png.png" width="30" height="30">
			</a>
				<a href="https://orcid.org/0000-0001-7277-9744">
			<img border="0" alt="OrcID" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/ORCID_iD.svg/768px-ORCID_iD.svg.png" width="30" height="30">
			</a>
				<a href="https://scholar.google.com/citations?user=6vpm6vcAAAAJ&hl=nl&oi=ao">
			<img border="0" alt="Google Scholar" src="https://www.pikpng.com/pngl/m/44-442505_google-scholar-twitter-instagram-google-scholar-icon-svg.png" width="30" height="30">
			</a>
				
				
				
			
			</div>
  			<div class="column right">
    			<h1>Research Projects</h1>
			
			<p><b>Development of single nucleus CUT&RUN (snCUT&RUN) to profile histone modification heterogeneity in HNSC</b></p>
				<p>Cancers are marked by extensive tumour heterogeneity. For example, different cancer types are marked by differences in driver genetic mutations and interactions with the microenvironment.
				Furthermore, different tumours of the same cancer type could already have significant differences, while within a tumour even several subclones could exhibit different genetic and epigenetic states.
				Tumour heterogeneity drives carcinogenesis and tumour evolution, and is hence a major obstacle in finding a cure for cancer.
				It is therefore essential that we understand tumour heterogeneity better. Relatively little is known about how epigenetic factors such as heterogeneous histone modification and methylation states can promote tumourigenicity and propensity to progress.
				Several tools have been develop to profile the methylome and histone modifications in single cells.
				However, studies that investigate epigenetic heterogeneity in cancers are still lacking.
				Hence, we developed snCUT&RUN, a method adapted from the CUT&RUN (cleavage under targets and release using nuclease) method originally developed in the Henikoff lab, to profile histone modifications H3K27ac and H3K4me3 in primary and progressed HNSC tumours at the single-cell level.
				With this project we seek to understand better the histone modification landscape of primary and progressed HNSC and whether epigenetic heterogeneity driven mechanisms could underlie the progression of HNSC tumours.
				This work has recently been published in Genome Research. </p>
			
				
			<p><b>Focal amplification mediated progression of squamous cancers</b></p>
				<p>A major genetic driver of tumour heterogeneity is chromosomal instability (CIN), leading to various driver and passenger copy number aberrations (CNAs). 
				CNAs of genes and chromosomes are frequently associated with increased tumorigenicity and propensity for cancers to progress into aggressive forms. 
				Focal amplifications (FA) are a type of CNA in which sections of chromosomes (typically a couple Mb in size) are amplified at high copy numbers.
				Recently, a number of studies have shown that FAs could occur either extrachromosomally, in form of circular DNA (ecDNA) typically 1-3Mb in size, or as homogeneously staining regions (HSRs).
				These studies elucidated amongst others that ecDNA/HSR-mediated FAs could promote tumour progression and therapy resistance. 
				Furthermore, ecDNA were shown to increase oncogene expression through increasing both gene copy number as well as activating regulatory elements such as enhancers.
				ecDNAs mediated novel interactions between the oncogenes and enhancers, further amplifying oncogene expression.
				Through research done on the prevalence of ecDNA across several cancer types, it is becoming increasingly clear that ecDNA is not prevalent in all cancer types. 
				Rather, other modes of FA such as intrachromosomal breakage-fusion-bridge (BFB) may be more prevalent in specific cancer types, such as lung, head and neck and cervical squamous cancers.
				In comparison to ecDNA research, studies on the biology of BFB-driven tumours remain lacking.
				Hence, this project aims to broaden the knowledge of the biology of BFB in promoting the progression of squamous cancers.
				Ultimately, this research could lead to the establishment of BFB mutations as potential biomarkers which can inform treatment responses of patients with these type of cancers.
				In this project, we utilize a broad range of techniques (genomics, epigenomics and cytogenetics) to profile BFB amplifications in squamous cancers.</p>
			
								
  			</div>
		</div>
				
		<footer>
    		<ul>
        		<li><a href="mailto:Daniel_Muliaditan@gis.a-star.edu.sg">E-mail</a></li>
        		<li><a href="https://github.com/dmuliaditan">Website</a></li>
			</ul>
		</footer>
	</body>
</html>
