<!DOCTYPE html>
<html>
	<head>
		<title>Welcome to my website</title>
		<!-- link to main stylesheet -->
		<link rel="stylesheet" type="text/css" href="/css/main.css">
		
				
		<style>
		* {
  		box-sizing: border-box;
			}

		/* Create two unequal columns that floats next to each other */
		.column {
  		float: left;
  		padding: 10px;
  		}

		.left {
  		width: 30%;
		}

		.right {
 		width: 70%;
		}

		/* Clear floats after the columns */
		.row:after {
  		content: "";
  		display: table;
  		clear: both;
		}
		
		img {
			border-radius: 50%;
		}
		

		</style>
	
	</head>
	<body>
		<nav>
    		<ul>
        		<li><a href="/">About</a></li>
	        	<li><a href="/research">Research</a></li>
        		<li><a href="/publications">Publications</a></li>
        		<li><a href="/cv">CV</a></li>
    		</ul>
		</nav>
		
		<div class="about-border"></div>
		
		<div class="row">
  			<div class="column left">
   		 	<p> </p>
				<p> </p>
				<p> </p>
				<img src="https://media-exp1.licdn.com/dms/image/C4D03AQEtXJfCa3lPEg/profile-displayphoto-shrink_800_800/0/1570023656793?e=1646265600&v=beta&t=BP19NBpE4RavFsZA-aXw02bW_SA7a2e9mqPpJHpPVN0" alt="Avatar" width="155" height="155">
  				
				<p> Daniel Muliaditan </p>
				<p> PhD-candidate Bioinformatics/Molecular Biology at NUS and GIS </p>
				
				<a href="https://www.linkedin.com/in/daniel-muliaditan-bb7769b2/">
			<img border="0" alt="LinkedIn" src="https://www.pngfind.com/pngs/m/53-534062_linkedin-for-lawyers-property-icon-circle-hd-png.png" width="30" height="30">
			</a> 
				<a href="https://twitter.com/dmuliaditan">
			<img border="0" alt="Twitter" src="https://www.pngfind.com/pngs/m/60-606593_icons-clipart-twitter-twitter-logo-png-transparent-png.png" width="30" height="30">
			</a>
				<a href="https://orcid.org/0000-0001-7277-9744">
			<img border="0" alt="OrcID" src="https://upload.wikimedia.org/wikipedia/commons/thumb/0/06/ORCID_iD.svg/768px-ORCID_iD.svg.png" width="30" height="30">
			</a>
				<a href="https://scholar.google.com/citations?user=6vpm6vcAAAAJ&hl=nl&oi=ao">
			<img border="0" alt="Google Scholar" src="https://www.pikpng.com/pngl/m/44-442505_google-scholar-twitter-instagram-google-scholar-icon-svg.png" width="30" height="30">
			</a>
				
				
				
			
			</div>
  			<div class="column right">
    			<h1>Research Projects</h1>
			<p><b>Karyotype heterogeneity-driven metastatic progression in HNSC</b></p>
				<p>Cancers are marked by extensive tumour heterogeneity. 
				For example, different cancer types are marked by differences in driver genetic mutations and interactions with the microenvironment.
				Furthermore, different tumours of the same cancer type could already have significant differences, while within a tumour even several subclones could exhibit different genetic and epigenetic states.
				Tumour heterogeneity drives carcinogenesis and tumour evolution, and is hence a major obstacle in finding a cure for cancer.
				It is therefore essential that we understand tumour heterogeneity better.
				A major genetic driver of tumour heterogeneity is chromosomal instability (CIN), leading to various driver and passenger copy number aberrations (CNAs). 
				CIN and CNAs were previously shown to be able to increase a tumour's fitness and resistance to therapy. 
				Profiling karyotypes of primary and progressed cancers therefore could reveal new biological understanding of how CNA landscape could promote tumour progression.
				In this project, we use the 10X Single Cell CNV solution to profile karyotypes of thousands of cells from primary and metastatic HNSC patient derived cancer cell lines.
				The overall aim of this project is to investigate how karyotypes of HNSC tumours change during metastasis and whether there are recurrent driver CNAs that enable increased metastatic potential of HNSC.</p>
				
			<p><b>Development of snCUT&RUN to profile histone modification heterogeneity in HNSC</b></p>
				<p>Apart from genetic heterogeneity, epigenetic factors such as heterogeneous histone modification and methylation states could also promote tumourigenicity and propensity to progress.
				Several tools have been develop to profile the methylome and histone modifications in single cells.
				However, studies that investigate epigenetic heterogeneity in cancers are still lacking.
				Hence, we're developing snCUT&RUN, a method adapted from the CUT&RUN (cleavage under targets and release using nuclease) method originally developed in the Henikoff lab, to profile histone modifications H3K27ac and H3K4me3 in primary and progressed HNSC tumours at the single-cell level.
				With this project we seek to understand better the histone modification landscape of primary and progressed HNSC and whether epigenetic heterogeneity driven mechanisms could underlie the progression of HNSC tumours.</p>
			
				
			<p><b>YAP1 amplification mediated novel cis- and trans-interactions in HNSC</b></p>
				<p>Copy number aberrations (CNAs) of genes and chromosomes are frequently observed in cancer and is associated with increased tumorigenicity and propensity for cancers to progress into aggressive forms. 
				Focal amplifications (FA) are a type of CNA in which sections of chromosomes (typically a couple Mb in size) are amplified at high copy numbers.
				Recently, a number of studies have shown that FAs could occur either extrachromosomally, in form of circular DNA (ecDNA) typically 1-3Mb in size, or as homogeneously staining regions (HSRs).
				These studies elucidated amongst others that ecDNA/HSR-mediated FAs could promote tumour progression and therapy resistance. 
				Furthermore, ecDNA were shown to increase oncogene expression through increasing both gene copy number as well as activating regulatory elements such as enhancers.
				ecDNAs mediated novel interactions between the oncogenes and enhancers, further amplifying oncogene expression.
				The extent of ecDNA presence across cancers is still not fully studied. Hence, this project aims to broaden the knowledge of ecDNA, focusing on YAP1 amplification in HNSC.
				YAP1 amplification is a common event in HNSC (~5% prevalence across HNSC patients, TCGA), and understanding whether YAP1 is amplified through ecDNA/HSR could lead to potential therapeutic strategies in these type of cancers.
				In this project, we utilize a broad range of techniques (WGS, FISH, CUT&RUN and HiChIP) to profile YAP1 amplification in HNSC, and to understand better the biological mechanisms behind ecDNA amplification and its role in shaping the cellular nucleus.</p>
			
								
  			</div>
		</div>
				
		<footer>
    		<ul>
        		<li><a href="mailto:Daniel_Muliaditan@gis.a-star.edu.sg">E-mail</a></li>
        		<li><a href="https://github.com/dmuliaditan">Website</a></li>
			</ul>
		</footer>
	</body>
</html>
